cpp/oneapi/dal/detail/error_messages.cpp

```diff
@@ -49,6 +52,12 @@ MSG(pushing_column_is_not_supported, "Pushing column is not supported")
 MSG(pushing_rows_is_not_supported_for_dpc, "Pushing rows is not supported for DPC++")
 MSG(pushing_rows_is_not_supported, "Pushing rows is not supported")
 MSG(unsupported_conversion_types, "Unsupported conversion types")
+MSG(number_of_rows_leq_zero, "Number of rows is lower than or equal to zero")
+MSG(number_of_columns_leq_zero, "Number of columns is lower than or equal to zero")
```

Comment:
We always use `row_count` and `column count`, so I'd suggest
```suggestion
MSG(table_rc_leq_zero, "Table row count is lower than or equal to zero")
MSG(table_cc_leq_zero, "Table column count is lower than or equal to zero")
```